### PR TITLE
docs: fix coding style by php-cs-fixer

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- run php-cs-fixer in #5765

**Checklist:**
- [x] Needs  #5774
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/general/helpers/005.php [hunk 1]
```diff
@@ -1 +1 @@
-<?= anchor('blog/comments', 'Click Here') ?>
+<?= anchor('blog/comments', 'Click Here');
```
Comment (kenjis) [inline user_guide_src/source/general/helpers/005.php:1]: The closing tag is deleted.

File: user_guide_src/source/outgoing/localization/005.php [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 namespace App\Controllers;
 
-class UserController extends \Controller
+use Controller;
```
Comment (kenjis) [inline user_guide_src/source/outgoing/localization/005.php:5]: Incorrect namespace.

File: user_guide_src/source/testing/database/002.php [hunk 3]
```diff
@@ -9,14 +9,14 @@ class MyTests extends CIUnitTestCase
 {
     use DatabaseTestTrait;
 
-    public function setUp()
+    protected function setUp()
```
Comment (kenjis) [inline user_guide_src/source/testing/database/002.php:12]: `: void` is missing.

File: user_guide_src/source/testing/database/002.php [hunk 4]
```diff
@@ -9,14 +9,14 @@ class MyTests extends CIUnitTestCase
...
     {
         parent::setUp();
 
         // Do something here....
     }
 
-    public function tearDown()
+    protected function tearDown()
```
Comment (kenjis) [inline user_guide_src/source/testing/database/002.php:19]: `: void` is missing.

File: user_guide_src/source/outgoing/localization/005.php (hunk 2 above)
Comment (kenjis) [reply user_guide_src/source/outgoing/localization/005.php:5]: Will be fixed in #5774

File: user_guide_src/source/testing/database/002.php (hunk 3 above)
Comment (kenjis) [reply user_guide_src/source/testing/database/002.php:12]: Will be fixed in #5774

File: user_guide_src/source/testing/database/002.php (hunk 4 above)
Comment (kenjis) [reply user_guide_src/source/testing/database/002.php:19]: Will be fixed in #5774

File: user_guide_src/source/general/helpers/005.php (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/general/helpers/005.php:1]: What shall we do?
Exclude this file?

Comment (paulbalandan) [reply user_guide_src/source/general/helpers/005.php:1]: I think add some dummy HTML

**Comment by sfadschm** [comment]
Only few comments, rest looks fine to me 👍

File: user_guide_src/source/helpers/form_helper/022.php [hunk 5]
```diff
@@ -11,7 +11,7 @@
 
 ?>
 
-<!-- Produces: --> 
+<!-- Produces: -->
```
[the remark this answers is not in the document]
Comment (sfadschm) [reply user_guide_src/source/helpers/form_helper/022.php:14]: Nevermind...

File: user_guide_src/source/database/query_builder/027.php [hunk 6]
```diff
@@ -2,9 +2,7 @@
 
 // With closure
 
-$builder->where('advance_amount <', function (BaseBuilder $builder) {
-    return $builder->select('MAX(advance_amount)', false)->from('orders')->where('id >', 2);
-});
+$builder->where('advance_amount <', static fn (BaseBuilder $builder) => $builder->select('MAX(advance_amount)', false)->from('orders')->where('id >', 2));
```
[the remark this answers is not in the document]
Comment (kenjis) [reply user_guide_src/source/database/query_builder/027.php:4]: Yes.

File: user_guide_src/source/helpers/text_helper/007.php [hunk 7]
```diff
@@ -1,8 +1,8 @@
 <?php
 
 $str = [
-    'question' => "Is your name O\'reilly?",
-    'answer'   => "No, my name is O\'connor."
+    'question' => "Is your name O\\'reilly?",
+    'answer'   => "No, my name is O\\'connor.",
```
[the remark this answers is not in the document]
Comment (kenjis) [reply user_guide_src/source/helpers/text_helper/007.php:5]: Yes. it does not show `O'reilly`, but `O\'reilly`.
These are data for `strip_slashes()`.

File: user_guide_src/source/general/helpers/005.php (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/general/helpers/005.php]: Added.

File: user_guide_src/source/database/query_builder/074.php [hunk 8]
```diff
@@ -2,14 +2,14 @@
...
-        ->orGroupStart()
-            ->where('b', 'b')
-            ->where('c', 'c')
-        ->groupEnd()
+    ->where('a', 'a')
+    ->orGroupStart()
+    ->where('b', 'b')
+    ->where('c', 'c')
+    ->groupEnd()
     ->groupEnd()
```
[the remark this answers is not in the document]
Comment (kenjis) [reply user_guide_src/source/database/query_builder/074.php:10]: Yes.

Disable `method_chaining_indentation` or exclude `database/query_builder/074.php`?

[the remark this answers is not in the document]
Comment (sfadschm) [reply user_guide_src/source/database/query_builder/074.php:10]: Oof, I think this is a very special case with multi-level chaining and the fixer cannot know which method returns which object, I guess.
So likely excluding.

[the remark this answers is not in the document]
Comment (kenjis) [reply user_guide_src/source/database/query_builder/074.php:10]: Okay, I added: https://github.com/codeigniter4/CodeIgniter4/pull/5765/commits/642dd8c9925bc13398ad6da57361c791b2a6a790

**Comment by kenjis** [comment]
@sfadschm @paulbalandan I've fixed all commented items.
Anything else?

**Comment by sfadschm** [comment]
I am through with all files 👍

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**